### PR TITLE
k11 pro encoder fixed

### PR DESCRIPTION
**Pull request by KeychronMacro**
Fixed the encoder interrupter part to solve the issue that encoder does not work when the backlight is in off status.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keychron/k11_pro/k11_pro.c [hunk 1]
```diff
@@ -135,8 +135,8 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
     return true;
 }
 
-#if defined(KC_BLUETOOTH_ENABLE) && defined(ENCODER_ENBALE)
-static void encoder0_pad_cb(void *param) {
+#if defined(KC_BLUETOOTH_ENABLE) && defined(ENCODER_ENABLE)
```
Comment (lokher) [inline keyboards/keychron/k11_pro/k11_pro.c]:
```suggestion
#if defined(ENCODER_ENABLE)
```

File: keyboards/keychron/q8_pro/q8_pro.c [hunk 2]
```diff
@@ -136,7 +136,7 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
 
 #if defined(KC_BLUETOOTH_ENABLE) && defined(ENCODER_ENABLE)
```
Comment (lokher) [inline keyboards/keychron/q8_pro/q8_pro.c]:
```suggestion
#if defined(ENCODER_ENABLE)
```

**Review by lokher — CHANGES_REQUESTED** [review]

File: keyboards/keychron/k11_pro/k11_pro.c (hunk 1 above)
Comment (KeychronMacro) [reply keyboards/keychron/k11_pro/k11_pro.c]: Changed

File: keyboards/keychron/q8_pro/q8_pro.c (hunk 2 above)
Comment (KeychronMacro) [reply keyboards/keychron/q8_pro/q8_pro.c]: Changed

**Pull request merged**